notSupported.md

```diff
@@ -0,0 +1,46 @@
+# Not Supported
+
+This is used to indicate the requested operation was not supported.
+
+## Problem Type
+
+type = https://problem.api.bunnings.com.au?type=notSupported
+
+## Specification
+
+The problem includes no additional properties on the base [RFC 7807](https://tools.ietf.org/html/rfc7807) problem.
+so uses the base specification
```

Comment:
This isn't correct, this is due to the RFC8087 not including in any support for the "errors" option. That also an I'm not sure what this comment relates to(see example vs specification statement also made)